mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -45,3 +46,33 @@ Feature: Requests tests
     When method POST
     Then status 422
     And match $.errors[0].message == 'Page requests are not allowed for this patron and item combination'
+
+  Scenario: Given an item Id, a user Id, and a pickup location, attempt to create a hold request when the applicable request policy disallows holds
+    * def userBarcode = 'FAT-1031UBC'
+    * def itemBarcode = 'FAT-1031IBC'
+    * def extRequestTypesForRequestPolicy = ["Page", "Recall"]
+
+    # post a material type and policies
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostPolicies') { extRequestTypes: #(extRequestTypesForRequestPolicy) }
+
+    # post an item
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostInstance')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostLocation')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostHoldings')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemId: #(itemId), extItemBarcode: #(itemBarcode) }
+
+    # post a group and an user
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostGroup')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserId: #(userId), extUserBarcode: #(userBarcode) }
+
+    # post a request and verify that the user are not allowed to create a page request
+    * def requestId = call uuid1
```

Comment:
Could we move it at the beginning of the test ? Please don't say me later I selected bunch of them. Just one line

Reply:
ok, fixed